u2f-tests/NFC/u2f_nfc_test.cc

```diff
@@ -168,11 +169,19 @@ int main(int argc, char* argv[]) {
   CHECK_EQ(0x6700u, xchgAPDUShort( 0, U2F_INS_REGISTER, 0, 0, 0, "", &rapduLen, rapdu));
   CHECK_EQ(0, rapduLen);
 
+  setChainingLc(256);
   cout << "Valid U2F_REGISTER, Short APDU\n";
   PASS(test_Enroll(SHORT_APDU, 0x9000u));
   cout << "Check the Signature\n";
   PASS(enrollCheckSignature( regReq , regRsp));
 
+  setChainingLc(100);
+  cout << "Valid U2F_REGISTER, Short APDU, Change BlockSize\n";
+  PASS(test_Enroll(SHORT_APDU, 0x9000u));
```

Comment:
it would be nice to pull constants like 0x900u and 0x6700u out, but perhaps not in this pull request.